### PR TITLE
SG-25955 - removing html tagging

### PR DESCRIPTION
**Pull request by shaynacohen**

### CODE REVIEW

**Comment by shaynacohen** [comment]
@NorberMV , are you OK with reviewing these fixes/changes?

File: docs/en/guides/pipeline-integrations/administration/file-system-config-reference.md [hunk 1]
```diff
@@ -105,29 +105,23 @@ Let's dive deeper into these modes.
...
-    <a name="this should be a list of dicts, each dict containing"></a>
     # this should be a list of dicts, each dict containing
-    <a name="three fields: path, relation and values"></a>
     # three fields: path, relation and values
-    <a name="(this is std {% include product %} API syntax)"></a>
     # (this is std {% include product %} API syntax)
-    <a name="any values starting with $ are resolved into path objects"></a>
     # any values starting with $ are resolved into path objects
     filters: [ { "path": "project", "relation": "is", "values": [ "$project" ] } ] 
+```
```
Comment (000paradox000) [inline docs/en/guides/pipeline-integrations/administration/file-system-config-reference.md:124]: Watching the preview, this snippet is better rendered when removing the quotes and the YAML label, like the snippet starting at line 140.

**Review by 000paradox000 — APPROVED** [review]
LGTM just small comment about snippet

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/42ecaa8b14b5ec2cd398066365985df1d74803e6/index.html)

**Review by NorberMV — APPROVED** [review]
LGTM!

File: docs/en/guides/pipeline-integrations/administration/file-system-config-reference.md (hunk 1 above)
Comment (000paradox000) [reply docs/en/guides/pipeline-integrations/administration/file-system-config-reference.md:124]: image

image

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/924701119a2874008d8376bc49eefc32bb5ce13d/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/74b33091fd4a7fc9270706279cec825950f993ba/index.html)

**Pull request merged**